pkgs/development/python-modules/stop-words/default.nix

```
@@ -0,0 +1,26 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+
+}:
+
+buildPythonPackage rec {
+  pname = "stop-words";
+  version = "2018.7.23";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-bfOtX13ml9qkN+REXIbHNgTmvBON0NwPrFVmSqTmsD4=";
+  };
+
+  pythonImportsCheck = [ "stop-words" ];
+
+  meta = {
+    description = "Get list of common stop words in various languages in Python.";
```

`meta.description` must not end with a period (see [Meta attributes](https://github.com/NixOS/nixpkgs/blob/5ff3f20fc8f72803d8ceb37af71e302887bc8df8/pkgs/README.md#meta-attributes))
```suggestion
    description = "Get list of common stop words in various languages in Python";
```